### PR TITLE
Add compatibility test for php.

### PR DESCRIPTION
**Pull request by TeBoring**

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: php/tests/compatibility_test.sh [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  PHP_CONFIG=`which php-config`
+  PHPIZE=`which phpize`
+  ln -sfn "/usr/local/php-${VERSION}/bin/php" $PHP
+  ln -sfn "/usr/local/php-${VERSION}/bin/php-config" $PHP_CONFIG
+  ln -sfn "/usr/local/php-${VERSION}/bin/phpize" $PHPIZE
+}
+
+generate_proto() {
+  PROTOC1=$1
+  PROTOC2=$1
```
Comment (xfxyjwf) [inline php/tests/compatibility_test.sh]: Should be $2 here?

**Comment by TeBoring** [comment]
See https://grpc-testing.appspot.com/me/my-views/view/All/job/protobuf_pull_request/1156/testReport/ for catching an API incompatible change.

**Comment by TeBoring** [comment]
Broken test not related to php.

**Pull request merged**